### DISCUSSION TITLE
Add issues to read list from series view

### DISCUSSION THREAD

**Discussion by morsok** [gotson/komga#1535] [Ideas]
### Describe your suggested feature

Imagine you recently discovered something like Vampirella and want to read them all from the beginning until the current series. You can either follow a reading order or decide to read them in the order the different series were published. You have to go inside each series, select all issues, and add to the read list, it's very time-consuming.

I'd like to be able from a series view (maybe the search result page?) to select multiple series and have the option to add all issues to a read list. Then you turn on the release date ordering and voila.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to the latest version.
- [X] I will fill out all of the requested information in this form.

**Comment by gotson** [comment]
I have made the "add to readlist" button in the multi-select bar for series, in which case it will add all the books of the series in the readlist.

**Reply by gotson** [reply]
Also added in the Series action menu !

**Comment by gotson** [comment]
🎉 This issue has been resolved in `1.19.0` ([Release Notes](https://github.com/gotson/komga/releases/tag/1.19.0))
